Assets/Scripts/Models/World.cs

```diff
@@ -156,10 +156,14 @@ void SetupWorld(int width, int height)
 
     public void Update(float deltaTime)
     {
-        foreach (Character c in characters)
+        for (int i = 0; i < characters.Count; i++)
         {
-            c.Update(deltaTime);
+            characters[i].Update(deltaTime);
         }
+        /*foreach (Character c in characters)
+        {
+            c.Update(deltaTime);
+        }*/
```

Comment:
How come this was changed? If there is a good reason to change it, please delete the commented out section.